src/main/java/org/opentripplanner/middleware/models/RelatedUser.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.middleware.models;
+
+/** A related user is a companion or observer requested by a dependent. */
+public class RelatedUser {
+    public enum RelatedUserStatus {
+        PENDING, CONFIRMED, INVALID
+    }
+
+    public String userId;
```

Comment:
I am pondering whether to let folks set nicknames for each companion they invite (e.g. "Grandma" "John Doe", "My Caregiver". Thoughts @br648 @josh-willis-arcadis ?

Reply:
I think that would be a nice feature to have. A nickname will look much cleaner on this UI than an email.
<img width="177" alt="image" src="https://github.com/user-attachments/assets/771f250b-34cb-4f90-83f6-d22b95b35afb">

Reply:
I agree. Will implement after reviews are complete.

Reply:
Do we need the dependent's name as well, to include in the email going to related users?

Reply:
Yeah, sounds like we need to add a `name` field to `OtpUser`.

Reply:
`name` added in: https://github.com/ibi-group/otp-middleware/pull/256/commits/581318cfc0b54048e886572b9f85ae291cf90533